### ISSUE TITLE
Filter Criteria Side Nav- 508 issue 

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#1491]
## Required Fix 

- [x] RF-4
- [x] RF-5a
- [x] RF-5c
- [x] RF-10
- [x] RF-15 

## Context 
See [EPA CAMD Accessibility Report ](https://teams.microsoft.com/l/file/31E8B6EF-1313-4495-B867-46347BB7E4A2?tenantId=88b378b3-6748-4867-acf9-76aacbeca6a7&fileType=docx&objectUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam%2FShared%20Documents%2FDatateers%2F5-UI%20Design%2F02%20508%20and%20Accessibility%2FEPA.CAMPD.Accessibility.Report%20.docx&baseUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam&serviceName=teams&threadId=19:f4349e2ba95e43f3b29e8208bc8f5d5f@thread.skype&groupId=21a5e1cd-d3f1-48a6-801f-0d34e2d63d23) for how to fix and additional details 

| Issue #  | Issue Description  |
|-|-|
| RF-4 | A modal dialogue appears that does not receive programmatic focus and is not immediately following that control that caused it to appear and/or focus is not returned when the dialog is closed. This issue is a violation of Section 508 and WCAG 2.0 Success Criterion 2.4.3 |
| RF-5a | Elements are present which are actionable by a mouse but not also actionable with simple keyboard controls (directional arrows, Tab, Enter, Space Bar). This issue is a violation of Section 508 and WCAG 2.0 Success Criterion 2.1.1. |
| RF-5c | For the filters Program, Unity Type, Unit Fuel Type, and Control Technology, you can tab to all the check boxes, but the content does not scroll. In other words, once you tab past the last visible checkbox on the page, you have no idea what you will be selecting. |
| RF-10 | There is informational content intended for screen reader users that is not intended to be displayed, but also is hidden because Display:none or aria hidden = true was used. This issue is a violation of Section 508 and WCAG 2.0 Success Criterion 1.3.1. |
| RF-15 | Elements on the page that have a state or property are not communicating the state or property to assistive technology. This issue is a violation of WCAG 2.0 Success Criterion 4.1.2 |

## Testing 
Tester: @aprematta

**Comment by aprematta** [comment]
Testing complete, please see results:

RF-4 - Most aspects fixed. Only issue is that the user is not trapped in the window until cancel or apply filter is selected. You can still get to the other content on the page.

RF-5a - Keyboard wise this looks good, I think a couple programatic issues were introduced:

ANDI is throwing that the form lacks an accessible name. 
All Close buttons are labeled close. I would actually say remove instead of close. So 'Remove selection for xxxxx"
Getting multiple ARIA errors in Axe

Recommend we go through on a call


RF-15 - Only one button has aria selected = true at a time. Aria selected = true should be set for all buttons where a filter selection has already been made.

**Comment by aprematta** [comment]
Testing complete, closing ticket
